src/components/forms/NetworkDevicesForm/NetworkDeviceIPAddress.tsx

```diff
@@ -0,0 +1,89 @@
+import { type FC } from "react";
+import type { InstanceAndProfileFormikProps } from "../instanceAndProfileFormValues";
+import { Input } from "@canonical/react-components";
+import { type LxdNetwork } from "types/network";
+import { type LxdNicDevice } from "types/device";
+import { typesWithNicStaticIPSupport } from "util/networks";
+
+interface Props {
+  formik: InstanceAndProfileFormikProps;
+  index: number;
+  network?: LxdNetwork;
+  device: LxdNicDevice;
+  readOnly: boolean;
+  family: "IPv4" | "IPv6";
+}
+
+export const NetworkDeviceIPAddress: FC<Props> = ({
+  formik,
+  index,
+  network,
+  device,
+  readOnly,
+  family,
+}) => {
+  if (!network || !network.config) {
+    return null;
+  }
+
+  const addressString = family === "IPv4" ? "ipv4.address" : "ipv6.address";
+
+  const getNetworkIP = () => {
+    return network.config[addressString];
+  };
```

Comment:
Can this and the callbacks below also be directly evaluated `const` strings to simplify?